### PR TITLE
Show default charts for variables, errors and parameters on calibrate run

### PR DESCRIPTION
**Pull request by jryu01**
# Description

On initial calibrate run, following charts will be selected by default

- variables that were selected in the mapping
- the error plots for those same variables and
- the parameter charts (only those that were not constant)

see: https://github.com/DARPA-ASKEM/terarium/issues/4863#issue-2537328926

https://github.com/user-attachments/assets/5dd7cd5d-f1fa-4fa0-9e38-35c67effbf9f

Resolves #(issue)

### CODE REVIEW

**Review by Tom-Szendrey — APPROVED** [review]
Thanks

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/services/model.ts [hunk 1]
```diff
@@ -208,6 +208,10 @@ export const getTypesFromModelParts = (model: Model) => {
 	return typeMapping;
 };
 
+export function getModelParameters(model: Model): ModelParameter[] {
+	return model?.semantics?.ode?.parameters ?? [];
+}
```
Comment (mwdchang) [inline packages/client/hmi-client/src/services/model.ts:213]: There are some overlapping here, to get parameters there is also
- model-representation/services#getParameters `https://github.com/DARPA-ASKEM/terarium/blob/921e3ee9e74950a5bc989dd6b2ced5b981315f55/packages/client/hmi-client/src/model-representation/service.ts#L104-L114`

CC @blanchco

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue [hunk 2]
```diff
@@ -1007,16 +1008,12 @@ async function getAutoMapping() {
...
-		modelPartTypes,
-		modelParameters: modelParams
-	} = await setupModelInput(modelConfigId.value);
+	const { model, modelConfiguration, modelOptions, modelPartUnits, modelPartTypes } = await setupModelInput(
+		modelConfigId.value
+	);
 	modelConfig.value = modelConfiguration;
 	modelStateOptions.value = modelOptions;
-	modelParameters.value = modelParams ?? [];
+	modelParameters.value = !model ? [] : getParameters(model);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue]: Could we instead improve `getParameters` to return `[]` if `model` is null or undefined.

Comment (mwdchang) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue]: I would argue no.

I think at some level, we need to have functions that act on concrete objects, if we constantly have to check if the parameters we pass in are null or undefined it becomes quite difficult to to write coherent code, or to debug problems once we have nested several levels of these type of logic.

File: packages/client/hmi-client/src/services/model.ts (hunk 1 above)
Comment (jryu01) [reply packages/client/hmi-client/src/services/model.ts]: I switched to use the existing function. Thanks!

**Pull request merged**